cassiopeia/tools/branch_length_estimator/IIDExponentialMLE.py

```diff
@@ -38,6 +40,10 @@ class IIDExponentialMLE(BranchLengthEstimator):
         minimum_branch_length: Estimated branch lengths will be constrained to
             have length at least this value. By default it is set to 0.01,
             since the MLE tends to collapse mutationless edges to length 0.
+        relative_mutation_rates: List of positive floats of length equal to the
+            number of character sites. Number at each character site indicates
+            the relative mutation rate at that site. Must be fully specified or
+            not at all.
```

Comment:
Instead of `Must be fully specified or not at all.`, say that it can be `None`, in which case all sites will be assumed to evolve at the same rate. Also say that `None` is the default value.